src/Tasks/Microsoft.NET.Build.Tasks/ProjectContext.cs

```diff
@@ -54,6 +56,15 @@ public IEnumerable<LockFileTargetLibrary> GetRuntimeLibraries(IEnumerable<string
                 allExclusionList.UnionWith(privateAssetsExclusionList);
             }
 
+            if(_filterlockFileTarget != null)
+            {
+                IEnumerable<LockFileTargetLibrary> filterLibraries = _filterlockFileTarget.Libraries;
+                Dictionary<string, LockFileTargetLibrary> filterLookup =
+                filterLibraries.ToDictionary(e => e.Name, StringComparer.OrdinalIgnoreCase);
```

Comment:
(nit) whitespace here.  This line should be indented since it is a continuation of the line above it.